python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -3140,6 +3140,33 @@ def AtenRoundFloatModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(5, 5, low = -3.0, high = 3.0))
 
 
+# ==============================================================================
+
+
+class AtenRoundClampFloatModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([5, 5], torch.float32, True),
+    ])
+    def forward(self, x):
+        # TOSA can only lower round when the dynamic range is restricted
+        # by a following clamp.
+        return torch.ops.aten.clamp(torch.ops.aten.round(x), -128, 127)
+
+
+@register_test_case(module_factory=lambda: AtenRoundClampFloatModule())
+def AtenRoundClampFloatModule_basic(module, tu: TestUtils):
+    module.forward(tu.rand(5, 5, low = -3.0, high = 3.0))
+
+
```

Comment:
Can you add another test which checks the float tensor + int addition case?

Reply:
will do